### PR TITLE
[OSDEV 562] Remove ppe fields

### PR DESCRIPTION
**Pull request by mazursasha1990**
[OSDEV 562](https://opensupplyhub.atlassian.net/browse/OSDEV-562) Remove PPE fields and indexing.

- Remove ppe fields from indexing processes 
- Remove ppe fields from API response

### CODE REVIEW

**Review by killev — COMMENTED** [review]
@mazursasha1990 could you please clearly describe what this PR does.
Please also add information about removed fields to release notes:
- release instructions
- Database changes

**Comment by mazursasha1990** [comment]
> @mazursasha1990 could you please clearly describe what this PR does. Please also add information about removed fields to release notes:
> 
> * release instructions
> * Database changes

Done

**Review by killev — COMMENTED** [review]
Please provide description to pull requests so that in can be understood in what is this about?

File: src/django/api/migrations/0130_introduce_separate_data_gathering_functions_for_the_index_table_columns.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 def introduce_separate_data_gathering_functions(apps, schema_editor):
     helper.run_sql_files([
         '0130_rename_facilityindexnew_forward.sql',
+        '0130_remove_ppe_fields_from_facilityindex.sql',
```
Comment (vladsha-dev) [inline src/django/api/migrations/0130_introduce_separate_data_gathering_functions_for_the_index_table_columns.py:14]: There should be a separate migration(new one) for removing the ppe fileds. We can't remigrate to 0130 migration.

File: src/django/api/models/facility/facility_index.py [hunk 2]
```diff
@@ -54,40 +54,6 @@ class FacilityIndex(models.Model):
         db_index=True,
         default=list,
         help_text='The related approved claim facilities.'))
-    ppe_product_types = postgres.ArrayField(
```
Comment (vladsha-dev) [inline src/django/api/models/facility/facility_index.py:57]: To properly remove these fields you should remove them via migration as well not only in the model file.

File: src/django/sqls/0130_remove_ppe_fields_from_facilityindex.sql
Comment (vladsha-dev) [inline src/django/sqls/0130_remove_ppe_fields_from_facilityindex.sql:1]: Following our convention, where for each new migration we create a new SQL file that includes a pointer to the related migration in its name, you should rename it.
E.g. 0136_remove_ppe_fields_from_facilityindex

**Review by killev — APPROVED** [review]
LGTM. Approved

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
I have discovered that ppe fields are used in numerous locations throughout the Django app, Dedupe Hub app, and React app. Please ensure that all mentions of them are deleted across all the apps and the necessary changes are made.

**Pull request closed without merging**